tests/api/test_account_permission_group.py

```diff
@@ -575,137 +838,37 @@ def test_permission_group_assign_users_mutation_customer_user(
-
-    group_staff_user = group.user_set.first()
-
-    variables = {
-        "id": graphene.Node.to_global_id("Group", group.id),
-        "users": [graphene.Node.to_global_id("User", group_staff_user.id)],
-    }
-
-    response = staff_api_client.post_graphql(query, variables)
-    assert_no_permission(response)
+    assert permission_group_data["permissions"] == []
 
 
-def test_permission_group_unassign_users_mutation_no_users_data(
+def test_group_delete_mutation_no_id(
     permission_group_manage_users, staff_user, permission_manage_staff, staff_api_client
 ):
     staff_user.user_permissions.add(permission_manage_staff)
-    group = permission_group_manage_users
-    query = PERMISSION_GROUP_UNASSIGN_USERS_MUTATION
-
-    variables = {
-        "id": graphene.Node.to_global_id("Group", group.id),
-        "users": [],
-    }
-
-    response = staff_api_client.post_graphql(query, variables)
-    content = get_graphql_content(response)
-    data = content["data"]["permissionGroupUnassignUsers"]
-    errors = data["accountErrors"]
-
-    assert errors
-    assert len(errors) == 1
-    assert errors[0]["field"] == "users"
-    assert errors[0]["code"] == AccountErrorCode.REQUIRED.name
-    assert data["group"] is None
-
-
-def test_permission_group_unassign_users_mutation_user_not_in_group(
-    permission_group_manage_users,
-    staff_user,
-    permission_manage_staff,
-    staff_api_client,
-    customer_user,
-):
-    staff_user.user_permissions.add(permission_manage_staff)
-    group = permission_group_manage_users
-    query = PERMISSION_GROUP_UNASSIGN_USERS_MUTATION
-
-    group_staff_user = group.user_set.first()
-
-    variables = {
-        "id": graphene.Node.to_global_id("Group", group.id),
-        "users": [graphene.Node.to_global_id("User", customer_user.id)],
-    }
+    query = PERMISSION_GROUP_DELETE_MUTATION
 
+    variables = {"id": None}
     response = staff_api_client.post_graphql(query, variables)
-    content = get_graphql_content(response)
-    data = content["data"]["permissionGroupUnassignUsers"]
-    errors = data["accountErrors"]
-    permission_group_data = data["group"]
-
-    assert errors == []
-    assert permission_group_data["name"] == group.name
-    assert permission_group_data["users"]
-    user_emails = {user_data["email"] for user_data in permission_group_data["users"]}
-    assert user_emails == {group_staff_user.email}
-    assert permission_group_data["permissions"]
-    result_permissions = {
-        permission["name"] for permission in permission_group_data["permissions"]
-    }
-    assert (
-        set(group.permissions.all().values_list("name", flat=True))
-        == result_permissions
-    )
-    permissions_codes = {
-        permission["code"].lower()
-        for permission in permission_group_data["permissions"]
-    }
-    assert (
-        set(group.permissions.all().values_list("codename", flat=True))
-        == permissions_codes
-    )
+    with pytest.raises(AssertionError):
```

Comment:
Check the message is the one excepted and not something else

Reply:
I'm not sure if we want to check error messages. @maarcingebala

Reply:
That would be better, otherwise you don't know for sure, it can be something else. It is often bad to check a very common exception without checking it is actually the proper and expected reason/trigger

Reply:
I removed that function because it checks only graphQL schema.